### ISSUE TITLE
Even the most basic functionality doesn't work

### ISSUE THREAD

**Issue by ChKaEs** [gamonoid/icehrm#281]
I copied all the files for v30 to my Godaddy CPanel. When I go to app/install, I fill in the information and click Test Database Connectivity, but then nothing else happens. No errors are displayed. Log file path is set to data/icehrm.log, but the file that actually gets created is named icehrm_install.log. 
This log file just has: 
[12-Dec-2021 19:38:00 UTC] Config writable 1
[12-Dec-2021 19:38:00 UTC] Config exists 1
[12-Dec-2021 19:46:05 UTC] isConfigFileExists 1
[12-Dec-2021 19:46:05 UTC] configData 
[12-Dec-2021 19:46:20 UTC] isConfigFileExists 1
[12-Dec-2021 19:46:20 UTC] configData 
[12-Dec-2021 19:46:22 UTC] isConfigFileExists 1
[12-Dec-2021 19:46:22 UTC] configData 
[12-Dec-2021 19:46:39 UTC] isConfigFileExists 1
[12-Dec-2021 19:46:39 UTC] configData 
[12-Dec-2021 20:07:11 UTC] isConfigFileExists 1
[12-Dec-2021 20:07:11 UTC] configData 
The Install Application button remains grey.

**Comment by ChKaEs** [comment]
I went through and did the manual installation, but when I attempt to go to the login page I get Virtual Class -- cannot instantiate.

**Comment by ChKaEs** [comment]
It seems the error may be related to the fact that I php 8 and iCE may not be compatible yet :(

**Comment by ChKaEs** [comment]
I set my php version to 7.4 just so I could test. I got the login page, but no matter what I tried it said login failed. When I check the log I am seeing tons of Access Denied errors for the user I create for the database. The user has all privileges.

**Comment by ChKaEs** [comment]
I still cannot even log in. I have looked through the installation sql files and did not find any reference to SystemData, so I am not sure what this error is about.
ERROR: (client=app) Error saving :: (Table 'icehrmdb.SystemData' doesn't exist) {"table":"SystemData","objectName":null,"keysToIgnore":["_table","_dbat","_tableat","_where","_saved","_lasterr","_original","foreignName","a","t"],"_dbat":"Company\\Common\\Model\\Timezones","_table":"SystemData","_tableat":"SystemData","_where":"name = ?","_saved":false,"_lasterr":"Model\\SystemData::UpdateActiveTable: Invalid table name: SystemData","_original":[],"foreignName":"model\\systemdata","name":"sysDataTime","value":1639779702} [] []

**Comment by ChKaEs** [comment]
I set my php version to 7.4 just so I could log in and test. When I went to the Employees, no employee was listed even though I can see a sample employee in the database table. I added a new employee, and again I can see this new employee in the database table, but the loading wheel on the iCE HRM Employees page just spins and no employees are dispalyed. If I move to other screens, it says no data, but the Employees screen just shows the spinning wheel.
